cloudflare_exporter/collector.py

```diff
@@ -177,9 +177,9 @@ def add(self, zone, serie):
 
 def _get_cloudflare_metrics_from_logs(token, logs_count, logs_sample, logs_range):
     cloudflare = CloudFlare.CloudFlare(debug=False, token=token)
-    # from cf docs: Must be at least 1 minute earlier than now and later than start
-    # Sometime 1 minutes is not enough...
-    end = int(time.time() - 60 * 1)
+    unixtime_lastminute = int(time.time() / 60) * 60
```

Comment:
(x / 60) * 60 == x? What does int casting bring?

Reply:
start at the minute  ( cloudflare always return a full minutes range ( from 0s to 59s )

Reply:
```
from datetime import datetime, timedelta
past_1minute = datetime.now() - timedelta(minutes=1)
unixtime = int().timestamp(past_1minute))
```
makes it more readable or at least more conventional (your intent is not ambiguous).

Reply:
yes better